editor/src/clj/editor/html_view.clj

```diff
@@ -239,41 +235,47 @@
     (str (http-server/url http-server)
          (resource/proj-path resource))))
 
-(defn- load-resource! [^WebEngine web-engine project resource]
-  (let [url         (resource-url (get-http-server! project) resource)
-        current-url (.getLocation web-engine)]
-    (when (not= url current-url)
-      (.load web-engine url))))
-
-(defn- update-web-view!
+(g/defnk load-resource! [^WebView web-view resource-node _node-id html]
   "Loads the url corresponding to the current resource.
 
   This will open the resource at the new location if we've moved it.
 
   If we somehow manage to browse away from the project to an external
   page, this will somewhat annoyingly bring us back."
-  [view-id project]
-  (let [web-view ^WebView (g/node-value view-id :web-view)
-        web-engine (.getEngine web-view)]
-    (when-let [resource-node (g/node-value view-id :resource-node)]
-      (load-resource! web-engine project (g/node-value resource-node :resource)))))
+  (when resource-node
```

Comment:
It seems to me that what was previously a docstring (`"Loads the url..."`) is now part of a function body. Maybe remove the string altogether?

Reply:
Oh, I missed that :flushed:. Thanks!